package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon --exec babel-node src/index.js",
+    "start": "nodemon --exec babel-node src/server.js",
```

Comment:
Nodemon should only be used in production. Also, remove babel-node, it's not allowed in production.

Reply:
Okay. Thanks for pointing that out. It's been updated